cli/ballerina-cli/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -1,94 +1,78 @@
 NAME
-       ballerina-run - Build and execute a Ballerina program
+       ballerina-run - Compile and run the current package
 
 SYNOPSIS
-       bal run 
-       bal run <executable-jar>
-       bal run [--target-dir] <target-directory-path>
-       bal run [--offline] [--experimental] [--debug] [--observability-included] 
-               <ballerina-file-path> 
-       bal run [--offline] [--experimental] [--debug] [--observability-included] 
-               <ballerina-file-path> [--] <args...>
-       bal run [--offline] [--experimental] [--debug] [--observability-included] 
-               <ballerina-package-path> [--] <args...>
+       bal run [OPTIONS] [<package>|<source-file>|<jar-file>] [--] <args...>
+
 
 DESCRIPTION
-       Build and execute the given standalone Ballerina file, package, or program.
+       Compile the current package and run it. This command does not produce
+       an executable program.
 
-       If a standalone Ballerina file or a package directory is given, this
-       command builds and runs it. Compilation is done internally and the
-       executable will not be created.
+       The run command executes the program when a jar file produced by 'bal build'
+       is given as the argument. If not, compiles and runs the given package
+       or single bal file.
```

Comment:
".bal file" maybe? To be consistent with the other places where we have mentioned this.

Reply:
We seem to have different usage including `.bal` and `'.bal'` also.

Reply:
+1